nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java

```diff
@@ -120,12 +120,28 @@ protected void setupArgline() {
                 this.originalArgLine + " " + this.configuration.toArgLine());
     }
 
-    private Xpp3Dom setReportOutputDirectory(Xpp3Dom configuration) {
+    private Xpp3Dom applyNonDexConfig(Xpp3Dom configuration) {
         Xpp3Dom configNode = configuration;
         if (configNode == null) {
             configNode = new Xpp3Dom("configuration");
         }
 
+        configNode = addExcludedGroups(configNode);
```

Comment:
consider making this something like:
`return addExcludedGroups(configNode).setReportOutputDirectory(configNode);`
This will appease Codacy and make the code shorter maybe :).